lib/android/app/src/main/java/com/reactnativenavigation/parse/FabOptions.java

```diff
@@ -25,38 +25,31 @@ public static FabOptions parse(JSONObject json) {
         options.id = TextParser.parse(json, "id");
         options.backgroundColor = ColorParser.parse(json, "backgroundColor");
         options.clickColor = ColorParser.parse(json, "clickColor");
-        options.rippleColor = ColorParser.parse(json, "rippleColor");
         options.visible = BoolParser.parse(json, "visible");
         if (json.has("icon")) {
             options.icon = TextParser.parse(json.optJSONObject("icon"), "uri");
         }
         options.iconColor = ColorParser.parse(json, "iconColor");
-        if (json.has("actions")) {
-            JSONArray fabsArray = json.optJSONArray("actions");
-            for (int i = 0; i < fabsArray.length(); i++) {
-                options.actionsArray.add(FabOptions.parse(fabsArray.optJSONObject(i)));
-            }
-        }
         options.alignHorizontally = TextParser.parse(json, "alignHorizontally");
-        options.alignVertically = TextParser.parse(json, "alignVertically");
         options.hideOnScroll = BoolParser.parse(json, "hideOnScroll");
         options.size = TextParser.parse(json, "size");
+        options.customSize = NumberParser.parse(json, "customSize");
+        options.useCompatPadding = BoolParser.parse(json, "useCompatPadding");
```

Comment:
I think we better remove this option and set it to true by default when creating the FAB.